example/index.html

```diff
@@ -156,7 +153,51 @@
          "<span style='color: black'><strong>Air Quality Index</strong></span>": AQICNLayer
 
     };
+    var allMapLayers = {"Baselayer1": baselayer1 ,
+    "Baselayer2": baselayer2 ,
+    "Baselayer3": baselayer3 ,
+    "Baselayer4": baselayer4 ,
+    "PurpleAirLayer-HeatMap": PurpleLayer ,
+    "PurpleAirMarkerLayer": PurpleAirMarkerLayer,
+    "SkyTruth": SkyTruth,
+    "Fractracker": Fractracker,
+    "ToxicRelease": ToxicRelease,
+    "OdorReport": OdorReport,
+    "MapKnitter": MapKnitter,
+    "OpenInfraMap_Power": OpenInfraMap_Power ,
+    "<span style='color: blue'><strong>OpenInfraMap_Telecom": OpenInfraMap_Telecom ,
+    "OpenInfraMap_Petroleum": OpenInfraMap_Petroleum ,
```

Comment:
I like that you've abstracted the layer names here. Do you think we could shorten the hash versions, like `OIMPower` and `JMAsian` so that they're more compact as a URL?